### ISSUE TITLE
Location of output data

### ISSUE THREAD

**Issue by 17100281** [khcs/learning-to-read#3]
Hello sir.  Could you please tell where will the output of training show?

**Comment by 17100281** [comment]
Or a better way to phrase my question is that could you please explain how to use your trained model to generate annotation for any new image that I have?

**Comment by khcs** [comment]
If you're just interested in using the provided pre-trained model to generate annotations for any images you have, you may want to see the [txtrnn/sample.lua](https://github.com/khcs/learning-to-read/blob/master/txtrnn/sample.lua) code.
Don't expect to get very good results, though. The BLEU scores on unseen test set are quite low as shown in the paper.
More work needs to be done to increase the performance, including collecting more (balanced) training data.
